pando/utils.py

```diff
@@ -17,6 +17,13 @@ def maybe_encode(s, codec='ascii'):
     return s.encode(codec) if isinstance(s, text_type) else s
 
 
+def try_encode(s, codec='ascii'):
+    try:
+        return s.encode(codec) if isinstance(s, text_type) else s
```

Comment:
This line duplicates `maybe_encode`. Seems like we should call that here, unless `maybe_encode` is now vestigial?